src/components/product.js

```diff
@@ -407,16 +407,20 @@ export default class Product extends Component {
    * @return {Object}
    */
   get trackingInfo() {
+    const info = {
+      destination: this.options.buttonDestination,
+    };
+
     if (this.selectedVariant) {
-      return {
+      return Object.assign({}, info, {
         id: this.id,
```

Comment:
Why include the empty object? If you just assign to info, there’s no need to return inside the if and this method just has one return stanza

Reply:
@minasmart valid observation; it's a matter of coding styles, but mostly best practices I guess, not mutating objects after creation brings a series of benefits, mostly related to avoiding unclear states.